packages/headless/src/controllers/smart-snippet-suggestions/headless-smart-snippet-suggestions.ts

```diff
@@ -0,0 +1,95 @@
+import {Engine} from '../../app/headless-engine';
+import {buildController, Controller} from '../controller/headless-controller';
+import {search} from '../../app/reducers';
+import {loadReducerError} from '../../utils/errors';
+import {SearchSection} from '../../state/state-sections';
+
+export interface SmartSnippetSuggestionsDocumentId {
```

Comment:
Could the QuestionAnswer "documentId" should have a specific interface that could be used here?

Reply:
Indeed, seems like it could be used [here](https://github.com/coveo/ui-kit/pull/897/files#diff-8263a1036b45564b97adfc1816f2e5a1c67c4e4da484cb024a2c0e298eefb50cR4).